pygeos/constructive.py

```diff
@@ -702,11 +698,11 @@ def segmentize(geometry, tolerance, **kwargs):
     --------
     >>> line = Geometry("LINESTRING (0 0, 0 10)")
     >>> segmentize(line, tolerance=5)
-    <pygeos.Geometry LINESTRING (0 0, 0 5, 0 10)>
+    <pygeos.Geometry LINESTRING (0 0, 0 5, 0 10)>  # doctest: +SKIP
```

Comment:
Reason for red CI; the skip needs to be on the line above (you did it correctly below)